app/controllers/user/teams_controller.rb

```diff
@@ -1,7 +1,45 @@
 class User::TeamsController < ApplicationController
+  before_action :require_user
+  
   def index
+    @teams = current_user.teams
   end
 
   def show
```

Comment:
We will need to prevent users from manually visiting other teams show page. In addition to `require_user`, this also needs to be tied to the `current_user`